boards/riscv32/rv32m1_vega/doc/index.rst

```diff
@@ -236,8 +236,8 @@ Pin   Port/bit   Function
 4     C/12       ADC0_SE7
 6     B/9        ADC0_SE3
 8     E/4        ADC0_SE21
-10    E/10       ADC0_SE19 (and E/10, I2C_SDA via 0 Ohm DNP)
-12    E/11       ADC0_SE20 (and E/11, I2C_SCL via 0 Ohm DNP)
+10    E/10       ADC0_SE19 (and E/10, I2C3_SDA via 0 Ohm DNP)
```

Comment:
These two look good, though; thanks! I generated these tables with macros from the schematic, and missed that this one doesn't have a number there.